### PR TITLE
Remove flashbang timer adjustment

### PR DESCRIPTION
**Pull request by Allspice**
whatever

### CODE REVIEW

**Comment by RealestEstate** [comment]
it was inevitable

**Comment by aceedex** [comment]
No don't

**Comment by ShiftyRail** [comment]
I'm not sure this is the reasonable course of action

**Comment by SonixApache** [comment]
great, now i have to carry manually made flashbangs

**Comment by iFlashYou** [comment]
The selfmade flashbangs only have the flash and no bang, afaik

**Comment by Diavill** [comment]
I can't be doin' like this man

**Comment by Accipitermaris** [comment]
3s minimum for flashbangs seems fine to me

**Comment by ShiftyRail** [comment]
I don't think the timer is that much of the issue
At any rate if you do that you'd need to update the flashbang box description

@Accipitermaris Turbo tried that compromise but nade code got him
If I'm not too depressed tonight I'll try to push a middle ground

File: code/game/objects/items/weapons/grenades/flashbang.dm [hunk 1]
```diff
@@ -123,6 +123,12 @@
 	..()
 	set_light(15)
 
+/obj/item/weapon/grenade/flashbang/attackby(obj/item/weapon/W as obj, mob/user as mob)
```
Comment (ShiftyRail) [inline code/game/objects/items/weapons/grenades/flashbang.dm]: The `as obj` and `as mob` are not needed.

**Comment by Chazach** [comment]
They targeted gamers.

Gamers.

We're a group of people who will sit for hours, days, even weeks on end performing some of the hardest, most mentally demanding tasks. Over, and over, and over all for nothing more than a little digital token saying we did.
We'll punish our selfs doing things others would consider torture, because we think it's fun.

We'll spend most if not all of our free time min maxing the stats of a fictional character all to draw out a single extra point of damage per second.


Do these people have any idea how many controllers have been smashed, systems over heated, disks and carts destroyed 8n frustration? All to latter be referred to as bragging rights?

These people honestly think this is a battle they can win? They take our media? We're already building a new one without them. They take our devs? Gamers aren't shy about throwing their money else where, or even making the games our selves. They think calling us racist, mysoginistic, rape apologists is going to change us? We've been called worse things by prepubescent 10 year olds with a shitty head set. They picked a fight against a group that's already grown desensitized to their strategies and methods. Who enjoy the battle of attrition they've threatened us with. Who take it as a challange when they tell us we no longer matter. Our obsession with proving we can after being told we can't is so deeply ingrained from years of dealing with big brothers/sisters and friends laughing at how pathetic we used to be that proving you people wrong has become a very real need; a honed reflex.

Gamers are competative, hard core, by nature. We love a challange. The worst thing you did in all of this was to challange us. You're not special, you're not original, you're not the first; this is just another boss fight.
damaged

**Comment by SonixApache** [comment]
Why can't you make flashbangs with chemistry, then?

**Comment by DamianX** [comment]
guess not
http://ss13.moe/index.php/poll/154

**Comment by GreenHairedAssistantRickysSmellyDick** [comment]
Fucking pathetic, little dick security mains who can't aim or robust so gotta spam instant entire screen wide stuns.

**Comment by SonixApache** [comment]
Fucking pathetic, little dick assistant mains who can't behave or follow the law so gotta spam instant entire github wide rants.

**Pull request closed without merging**